in_toto/verifylib.py

```diff
@@ -1119,6 +1123,9 @@ def verify_item_rules(source_name, source_type, rules, links):
   source_materials_queue = list(source_materials.keys())
   source_products_queue = list(source_products.keys())
 
+  # Reset rule traceback dictionary
+  RULE_TRACE.clear()
```

Comment:
Can you mention clearing and populating the global `RULE_TRACE` in the docstring's side effects section?